### PR TITLE
[Kotlin Migration ] MathJaxClozeTest

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin migration of MathJaxClozeTest .

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/libanki/MathJaxClozeTest.kt [hunk 1]
```diff
@@ -1,126 +1,105 @@
...
-
-        assertThat(cards.get(0).q(), containsString("class=cloze"));
-        assertThat(cards.get(1).q(), containsString("class=cloze"));
-        String s = cards.get(2).q();
-        assertThat(cards.get(2).q(), not(containsString("class=cloze")));
-        assertThat(cards.get(3).q(), containsString("class=cloze"));
-        assertThat(cards.get(4).q(), containsString("class=cloze"));
+    fun verifyMathJaxClozeCards() {
+        val c = col
+        val f = c.newNote(c.models.byName("Cloze"))
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/libanki/MathJaxClozeTest.kt:37]: ad a newline around this like it was in the Java

File: AnkiDroid/src/test/java/com/ichi2/libanki/MathJaxClozeTest.kt [hunk 2]
```diff
@@ -1,126 +1,105 @@
...
+import com.ichi2.libanki.template.MathJax
+import com.ichi2.libanki.template.TemplateFilters
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.*
+import org.junit.Assert.*
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MathJaxClozeTest : RobolectricTest() {
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/libanki/MathJaxClozeTest.kt:16]: cleanup: rename `f` to something usable

We no longer have the type to suggest what it is

**Review by david-allison — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/test/java/com/ichi2/libanki/MathJaxClozeTest.kt (hunk 2 above)
Comment (mikehardy) [reply AnkiDroid/src/test/java/com/ichi2/libanki/MathJaxClozeTest.kt:16]: done directly

**Review by mikehardy — APPROVED** [review]
LGTM

**Review by david-allison — APPROVED** [review]

**Pull request merged**